v1/transactions/transactions.gunk

```diff
@@ -196,6 +196,63 @@ type CreateTransactionResponse struct {
 	CratedAt string `pb:"2" json:"created_at"`
 }
 
+// TFAType is available type of TFA.
+type TFAType int
+
+const(
+	_ TFAType = iota
+
+	// SMS Message.
+	SMS
+
+	// SAFEKEY Application.
+	SAFEKEY
+)
+
+// ApprovePaymentRequest wraps all request to request approval
+// of pending transaction.
+type ApprovePaymentRequest struct{
+	// TransactionID is the transaction unique identifier.
+	TransactionID string `pb:"1" json:"transaction_id"`
+
+	// Approve is Boolean value of approval action.
+	Approve bool `pb:"2" json:"approve"`
+
+	// TFAType is type to receive OTP authentication code.
+	TFAType TFAType `pb:"3" json:"tfa_type"`
+}
+
+// ApprovePaymentResponse response for approve payment request.
+type ApprovePaymentResponse struct{
+	// AuthorizationID is the executable code is obtained from
+	// the payment feedback result
```

Comment:
I kinda don't understand this line

Reply:
i got this from ACB proto definition, not sure what that mean. I will ask Cuong later on.